script/Arcueid/Day 8/08_00_ARC08_8_2.txt

```diff
@@ -246,6 +247,7 @@ And not even a very serious question at that.
 [sha:6fdfb403ceb9f85866603e5ce884daada1fe977c]{
 -- Page 598, Offset 21287.
 -- なに、コドモのくせにカマかけてきたワケ？　そういうところはナマイキで、先生、キライだな！」
+// group pr note:change brat?
```

Comment:
What, even though you're just some brat you're trying to ask %{i}me%{/i} leading questions?